src/scripts/site.js

```diff
@@ -831,7 +829,6 @@ var AppInfo = {};
         define("viewSettings", [componentsPath + "/viewsettings/viewsettings"], returnFirstDependency);
         define("filterMenu", [componentsPath + "/filtermenu/filtermenu"], returnFirstDependency);
         define("sortMenu", [componentsPath + "/sortmenu/sortmenu"], returnFirstDependency);
-        define("connectionmanager", [apiClientBowerPath + "/connectionmanager"]);
```

Comment:
Should this be kept now?

Reply:
connectionmanager does nothing, connectionManager is the module that is used everywhere. connectionManager is defined after instantiate the connectionManagerFactory module.